Sources/CubeCore/CubeSceneView.swift

```diff
@@ -146,8 +146,8 @@ public class CubeSceneView: SCNView, UIGestureRecognizerDelegate {
           let cube = CubeNode(
             letterGeometry: letterGeometry,
             store: store
+              .scope(state: \.cubes[index], action: { $0 })
               .actionless
```

Comment:
For consistency should we use `absurd` here too, or should we use `.actionless` above?